examples/ex4_0.ipynb

```diff
@@ -721,7 +721,7 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "In this sample the conditional probability `P(sez_noodle|has_butter)` is approximately 1:2, while `P(has_butter|sez_noodle)` is approximately 1:4 (i.e., significantly lower).\n",
+    "We added (has_butter, sez_noodle) tuples. In this sample the conditional probability `P(sez_noodle|has_butter)` is approximately 1:2, while `P(has_butter|sez_noodle)` is approximately 1:4 (i.e., significantly lower).\n",
```

Comment:
I'm getting the opposite conditional probas, it doesn't change the conclusion that those rules are good.

```
Counter({'[True, False]': 53,
         '[False, False]': 27,
         '[True, True]': 7,
         '[False, True]': 15})
```

P(sez_noodle|has_butter) = P(sez_noodle and has_butter) / P(has_butter)
= 7 / (7 + 53)
~ 0.12 = 1/8